src/extras/gizmo/tri-data.js

```diff
@@ -0,0 +1,74 @@
+import { Vec3 } from '../../core/math/vec3.js';
+import { Quat } from '../../core/math/quat.js';
+import { Mat4 } from '../../core/math/mat4.js';
+import { Tri } from '../../core/shape/tri.js';
+import { Geometry } from '../../scene/geometry/geometry.js';
+
+// temporary variables
+const tmpV1 = new Vec3();
+const tmpV2 = new Vec3();
+const tmpV3 = new Vec3();
+
+/**
+ * The class for holding triangle data.
+ *
+ * @ignore
+ */
+class TriData {
+    /**
+     * The priority of the triangle data (Used for intersection ordering).
+     * priority = 0 - no priority
+     * priority > 0 - higher value represents a higher priority
```

Comment:
I know this isn't API, but if this was, this wouldn't render right. You probably want bullets here. Otherwise, it'd all be on the same line.